src/main/java/ATMaccount.java

```diff
@@ -0,0 +1,42 @@
+package src.main.java;
+
+public class ATMaccount {
+	private float balance;
+	private User user;
+	private boolean exit = false;
+	
+	public ATMaccount(User user) {
+		this.user = user;
+	}
+	
+	public User getUser() {
+		return user;
+	}
+	
+	public float CheckBalance() {
```

Comment:
Method names and variables should respect the [camel case](https://www.javatpoint.com/camel-case-in-java) convention. I suggest to apply this and refactor this method's name.

Reply:
right,I just didn't have the habit to do that